db/seeds.rb

```diff
@@ -7,3 +7,6 @@
 #
 #   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
 #   Character.create(name: 'Luke', movie: movies.first)
+
+Norm.create(name: 'full_time')
+Norm.create(name: 'part_time')
```

Comment:
If I run `db:seed` twice, will these value be created twice?

The expectation here would be that no matter how many times `db:seed` is ran, the data would only be created once.